floor_annotation/pubspec.lock

```diff
@@ -5,7 +5,7 @@ packages:
     dependency: "direct main"
     description:
       name: meta
-      url: "https://pub.dartlang.org"
+      url: "https://pub.flutter-io.cn"
```

Comment:
❓ Is there a way for us to keep referring to `pub.dartlang.org` for dependencies even when developing from China?

Reply:
sorry to change pub.dartlang.org，in my repository，the pubspec.lock is not on version control system，i forget to check this file